File: 0062-unique-paths/0062-unique-paths.cpp
class Solution {
public:
    int uniquePaths(int m, int n) {

        // int r = m-1;
        // double res = 1;
        // for(int i=1;i<=r;i++){
        //     res*=(n-1+i)/i;
        // }

        // return (int)res;

        int r = m-1;
	    double res = 1;
        for(int i=1;i<=r;i++){
            res = res * (n-1+i)/i;
        }

        return (int)res;
        
    }
};

File: 0062-unique-paths/0062-unique-paths.java
class Solution {
    public int uniquePaths(int m, int n) {
        int N = m+n-2;
        int r = m-1;
        double res = 1;
        for(int i=0;i<r;i++){
            res*= (N-i);
            res/=i+1;
            System.out.println(res);
        }

        return (int)Math.round(res);
        
    }
}